src/dune_rules/ml_sources.ml

```diff
@@ -413,7 +435,8 @@ let modules_of_stanzas dune_file ~dir ~scope ~lookup_vlib ~modules =
   >>| filter_partition_map
 
 let make dune_file ~dir ~scope ~lib_config ~loc ~lookup_vlib
-    ~include_subdirs:(_loc, (include_subdirs : Dune_file.Include_subdirs.t))
+    ~include_subdirs:
```

Comment:
is this pattern destructured for readability? we could just pass `~include_subdirs` as we get it

Reply:
We're also matching on `include_subdirs` in this function.

Reply:
indeed, should have expanded more